tests/cli/test_compose_sanity.sh

```diff
@@ -22,6 +22,30 @@ rlJournalStart
     rlPhaseEnd
 
     rlPhaseStartTest "compose start"
+        UUID=`$CLI compose start example-http-server ami`
+        rlAssertEquals "exit code should be zero" $? 0
+        UUID=`echo $UUID | cut -f 2 -d' '`
+
+        #wait for the build to properly start
+        sleep 20
+    rlPhaseEnd
+
+    rlPhaseStartTest "compose info"
+        if [ -n "$UUID" ]; then
+            rlRun -t -c "$CLI compose info $UUID | egrep 'RUNNING|WAITING'"
+        else
+            rlFail "Compose UUID is empty!"
+        fi
+    rlPhaseEnd
+
+    rlPhaseStartTest "cancel compose"
+        `$CLI compose cancel $UUID`
+        `$CLI compose status | grep $UUID`
+        rlAssertNotEquals "compose is canceled" $? 0
+        rlAssertNotExists "/var/lib/lorax/composer/results/$UUID/"
```

Comment:
We're going to have to remove the last assert when migrating to the osbuild-composer backend so better to remove it now. Just assert on current behavior which is that the compose UUID is nowhere to be found and the exit codes of the commands above. That is sufficient to verify the original bug which this test covers.

Reply:
Right, makes sense, so I'm just checking that that the compose UUID is not there when running composer-cli status now.